index.html

```diff
@@ -2560,7 +2560,12 @@ <h2 id="OfflineAudioCompletionEvent">
                 The <code>destination</code> parameter is the
                 <a><code>AudioParam</code></a> to connect to. This method does
                 not return <code>destination</code>
-                <a><code>AudioParam</code></a> object.
+                <a><code>AudioParam</code></a> object. <span class=
+                "synchronous">If <var>destination</var> belongs to an
```

Comment:
I'm defining this properly in a branch I'm doing to define the AudioNode ctors and other ctors, but I agree this is a bit lose for now.